.npmignore

```diff
@@ -0,0 +1,44 @@
+# Logs
```

Comment:
They've removed it already 🙂
https://github.com/typings/generator-typings/commit/582f26e3a295064834a13a83d899f9731956a32e